dem/getting-started/quanta-solution.md

```diff
@@ -0,0 +1,26 @@
+---
+id: quanta-solution
+title: The Quanta Solution
+---
+
+# The Quanta Solution
+
+**Quanta is a Digital Experience Monitoring platform designed to combine web performance and digital sustainability.** It enables tech, product, and business teams to collaborate efficiently to deliver fast and reliable digital experiences while saving hardware resources.
+
+The solution combines several complementary components:
+
+- **Synthetic Monitoring**: Simulates user journeys via web browsers to measure loading times (notably through Core Web Vitals) and detect slowdowns or malfunctions.
+- **Real User Monitoring (RUM)**: Detailed analysis of the actual experience of visitors on the application.
+- **Digital Sustainability Monitoring**: Continuous assessment of the carbon footprint and eco-design score of each page.
+- **Load Testing**: Validates the technical robustness of the site under high traffic conditions.
+
+Quanta provides a clear and unified interface, enhanced by:
+
+- **Dynamic graphs** to visualize loading times,
+- **Customizable and intelligent alerts**,
+- **Comprehensive audit reports**,
+- And human support from our experts (the *Perfgurus*), to identify the most impactful areas for improvement.
+
+**Objective: to help you improve user experience, boost your SEO performance, secure your traffic peaks, and concretely reduce your environmental impact.**
+
+Watch the [video presentation of the Quanta interface](https://www.youtube.com/watch?v=ae_8gF-KFG8).
```

Comment:
Video is in french, either we have an english version or we shouldn't link it in english doc